src/WorkspaceSystem/DynamicWorkspaceItem.vala

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * SPDX-FileCopyrightText: 2025 elementary, Inc. (https://elementary.io)
+ */
+
+public class Dock.DynamicWorkspaceIcon : BaseItem {
+    class construct {
+        set_css_name ("icongroup");
+    }
+
+    public DynamicWorkspaceIcon () {
```

Comment:
This can be removed I think? Or do you want it to be more explicit? Would break with convention though :shrug:

Reply:
Compilation fails with:
```
../src/WorkspaceSystem/DynamicWorkspaceItem.vala:6.1-6.49: error: unable to chain up to private base constructor
    6 | public class Dock.DynamicWorkspaceIcon : BaseItem {
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
Compilation failed: 1 error(s), 0 warning(s)
ninja: build stopped: subcommand failed.
```